README.md

```diff
@@ -56,9 +56,12 @@ defmodule MyApp.PostView do
 end
 ```
 
-is an example of a basic view. You can now call `render(conn, MyApp.PostView, "show.json", %{data: my_data, meta: meta})` or `'index.json` normally.
+is an example of a basic view. You can now call
```

Comment:
looks like this text got cut off when viewing w/ markdown

Reply:
Fixed!